Project.toml

```diff
@@ -1,20 +1,10 @@
 name = "AutoHashEqualsCached"
 uuid = "887ad303-ad3a-4bc5-965e-507c643d6945"
 authors = ["Neal Gafter <neal@gafter.com> and contributors"]
-version = "0.3.0"
+version = "1.0.0"
```

Comment:
do we want to leap straight to 1.0? or maintain a bit easier breakage for the time being?

Reply:
First, this is a breaking change because it changes how one specifies the hash function, and also because it no longer exports `@auto_hash_equals_cached`.

Second, I believe this is the functionality we should be placing into `AutoHashEquals.jl`, so this is a step in that direction.  We need to bite that bullet eventually to keep the macro interface as simple as possible.  Once I've done that, I'll update `raicode`.  Since this pacakge isn't registered, I doubt there are other users.